src/components/Capture/Face.js

```diff
@@ -25,6 +25,9 @@ class Face extends Component {
   static defaultProps = {
     useWebcam: true,
     requestedVariant: 'standard',
+    /* NOTE: This should be set to false when releasing! It is set to true in order to make it easier to test in PR form */
+    useMultipleSelfieCapture: true,
```

Comment:
You can default this to `false` and receive this option through a query string in our demo app